### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- absl-py = ">=0.9,<1.4"
- rocketchat_API = ">=0.6.31,<1.28.0"
- pytest-cov = "^4.0.0"
- moto = "~=4.0.8"
- towncrier = "^22.8.0"

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -157,7 +157,7 @@ flake8-docstrings = "^1.5.0"
 google-cloud-storage = "^2.4.0"
 azure-storage-blob = "<12.12.0"
 coveralls = "^3.0.1"
-towncrier = "^21.3.0"
+towncrier = "^22.8.0"
```
Comment (ancalita) [inline pyproject.toml:160]: I would double-check if we don't need to update some CLI towncrier commands we use in the release script with this major upgrade.

Comment (sanchariGr) [reply pyproject.toml:160]: Ran `towncrier build --yes --version 0.0.0` which is in the release script and `towncrier --draft` which is mentioned in the readme and both worked.

Comment (ancalita) [reply pyproject.toml:160]: Perfect, thank you!

**Review by ancalita — APPROVED** [review]

**Comment by sanchariGr** [comment]
@ancalita theres also an aiogram upgrade https://github.com/RasaHQ/rasa/pull/11646, but i remember you mentioning some issue recently with aiogram so wanted to double check if this can be upgraded?

**Comment by ancalita** [comment]
> @ancalita theres also an aiogram upgrade #11646, but i remember you mentioning some issue recently with aiogram so wanted to double check if this can be upgraded?

There's no issue with aiogram, just a tiny bug in the telegram channel implementation (ATO-492). It should be fine to upgrade the dependency.

**Pull request merged**